python/pylibcugraph/pylibcugraph/tests/test_node2vec.py

```diff
@@ -0,0 +1,131 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import cupy as cp
+import numpy as np
+
+
+# =============================================================================
+# Test data
+# =============================================================================
+# The result names correspond to the datasets defined in conftest.py
+# Note: the only deterministic path(s) in the following datasets
+# are contained in Simple_1
+_test_data = {"karate.csv": {
+                  "seeds": cp.asarray([0, 0], dtype=np.int32),
+                  "paths": cp.asarray([0, 8, 33, 29, 26, 0, 1, 3, 13, 33],
+                                      dtype=np.int32),
+                  "weights": cp.asarray([1., 1., 1., 1., 1., 1., 1., 1.],
+                                        dtype=np.float32),
+                  "path_sizes": cp.asarray([5, 5], dtype=np.int32),
+                  "max_depth": 5
+                  },
+              "dolphins.csv": {
+                  "seeds": cp.asarray([11], dtype=np.int32),
+                  "paths": cp.asarray([11, 51, 11, 51],
+                                      dtype=np.int32),
+                  "weights": cp.asarray([1., 1., 1.],
+                                        dtype=np.float32),
+                  "path_sizes": cp.asarray([4], dtype=np.int32),
+                  "max_depth": 4
+                  },
+              "Simple_1": {
+                  "seeds": cp.asarray([0, 3], dtype=np.int32),
+                  "paths": cp.asarray([0, 1, 2, 3],
+                                      dtype=np.int32),
+                  "weights": cp.asarray([1., 1.],
+                                        dtype=np.float32),
+                  "path_sizes": cp.asarray([3, 1], dtype=np.int32),
+                  "max_depth": 3
+                  },
+              "Simple_2": {
+                  "seeds": cp.asarray([0, 3], dtype=np.int32),
+                  "paths": cp.asarray([0, 1, 3, 5, 3, 5],
+                                      dtype=np.int32),
+                  "weights": cp.asarray([0.1, 2.1, 7.2, 7.2],
+                                        dtype=np.float32),
+                  "path_sizes": cp.asarray([4, 2], dtype=np.int32),
+                  "max_depth": 4
+                  },
+              }
+
+# =============================================================================
+# Pytest fixtures
+# =============================================================================
+# fixtures used in this test module are defined in conftest.py
+
+
+# =============================================================================
+# Tests
+# =============================================================================
+
+@pytest.mark.parametrize("compress_result", [True, False])
+def test_node2vec(sg_graph_objs, compress_result):
```

Comment:
How is this working. I don't see you creating the fixture `sg_graph_objs` in `pylibcugraph/tests/conftest.py` unless I am missing something